### DISCUSSION TITLE
The "utils" repo should have a more distinct name

### DISCUSSION THREAD

**Discussion by AnthonyGiorgio** [zopencommunity/meta#52] [General]
Since the `utils` repo is the core of `zopen` functionality, it should probably have a more distinct name.  Having a directory named `$HOME/utils` doesn't tell me what's in it at first glance.  Something like `zopen-utils` or `utils-zopen` would be more descriptive.

**Comment by IgorTodorovskiIBM** [comment]
I share similar concerns with the 'utils' repo name. I'd prefer zopen-utils (or even zopen) because I have other dirs starting with util compared to zopen and I prefer my tab autocompletion to be quick.

**Reply by MikeFultonDev** [reply]
Makes sense. If we do the change should we create zopen meta too? My preference would be zopenutils without the hyphen just for consistency with the other repos?

**Comment by IgorTodorovskiIBM** [comment]
Upvote if you prefer zopen-utils and zopen-meta

**Comment by IgorTodorovskiIBM** [comment]
Upvote if you prefer zopenutils and zopenmeta

**Comment by IgorTodorovskiIBM** [comment]
Upvote if you prefer just utils and meta

**Comment by IgorTodorovskiIBM** [comment]
Upvote if you prefer utils-zopen and meta-zopen

**Comment by s66perry** [comment]
Isn't the dir `$HOME/zopen/utils`?  And if not that should be the starting point.  I'm fine with renaming and open to any options.  The only real requirements for me are:
- these should be under `$HOME/zopen`.  Put everything related to these tools under the same top level dir.
- don't repeat zopen (i.e. $HOME/zopen/zopen-utils)
- don't rename utils to core :)

**Reply by AnthonyGiorgio** [reply]
You know, I never thought of putting this under the `zopen` directory. That's a really good idea!

**Reply by IgorTodorovskiIBM** [reply]
That works, one other consideration is that we could have one monorepo named zopen which would hold utils, meta, and potentially others.

**Reply by IgorTodorovskiIBM** [reply]
Expanding on the monorepo idea:
* A zopen repo which would hold utils and meta under one umbrella.
  * A side benefit of this is that it can set up the directory structure just by cloning zopen into $HOME.
* The repo would ignore files and directories inside boot and prod dirs (via .gitignore) so that they're not tracked by git
  * This would allow you to use zopen download to download into prod and boot and not affect the git repo
* The zopen repo would be a natural home for discussions, general issues, etc.

**Comment by dougburns** [comment]
There's no reason that you have to clone into a directory that matches the github repo name, so everyone can pick their preferred name.
```git clone repo-name folder-name```
